src/panels/lovelace/hui-root.ts

```diff
@@ -117,6 +118,8 @@ class HUIRoot extends LitElement {
   }
 
   protected render(): TemplateResult | void {
+    const isRTL = computeRTL(this.hass);
```

Comment:
Don't calculate it here, as that means that we will calculate it on each render, including when not in edit mode (99% of the time).